src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc

```diff
@@ -59,14 +61,14 @@ CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT=che-ws-__<username>__
 
 [IMPORTANT]
 ====
-Using the _One namespace per user_ or _One namespace for all workspaces_ strategy with the `common` PVC
-strategy requires persistent volumes with `ReadWriteMany` access mode if more than 1 workspace needs to be running
-concurrently. Because this might be tricky to set up reliably you may want to use a different PVC strategy or, in case
-of _One namespace per user_, to limit the number of concurrently running workspaces per user to 1.
+To run more than one workspace at a time when using this strategy together with the `common` PVC strategy, configure persistent volumes to use `ReadWriteMany` access mode.
 
-To enforce this limit, set:
+To limit the number of concurrently running workspaces per user to one, set the `CHE_LIMITS_USER_WORKSPACES_RUN_COUNT` environment variable to `1`.
 
-----
-CHE_LIMITS_USER_WORKSPACES_RUN_COUNT=1
-----
+// Links to Helm and Operator docs need to be added below:
+
+To limit the number of concurrently running workspaces per user to one (1):
+
+* For Helm Chart deployments: set the `.global.workspace.number` parameter to `1`.
+* For Operator deployments: set the `spec.server.cheCustomProperties.CHE_LIMITS_USER_WORKSPACE_RUN_COUNT` variable of the CheCluster Custom Resource (RC) to `1`.
```

Comment:
Is `RC` the shortcut we're using in the docs for a custom resource? I'm usually seeing `CR` in the wild.

Reply:
No, that's just my typu :/ Fixing now.